### ISSUE TITLE
magento2.0 (stable version) sample data install  issue

### ISSUE THREAD

**Issue by sivajik34** [magento/magento2#2400]
at the time of Module 'Magento_ConfigurableSampleData': process i'm getting following error
[Magento\Framework\Exception\SessionException]  
  Area code not set: Area code must be set before starting a session.
[Magento\Framework\Exception\LocalizedException]  
  Area code is not set

**Comment by andrew-smart** [comment]
I also have issues installing the sample data.

> Something went wrong while installing sample data. Please check var/log/system.log for details. You can retry installing the data now or just start using Magento.

And in the var/log/system.log

> [2015-11-18 09:29:33] main.ERROR: Sample Data error: Error in data structure: entity values are mixed [] []
> [2015-11-18 09:29:35] main.ERROR: Sample Data error: Validation is failed. [] []

Looking through the admin though a lot of the sample data has been installed.

**Comment by voleye** [comment]
Hi @sivajik34 thank you for reporting the issue!

I came across the problem before but I can not reproduce it for now.

It may be caused by the cache if you used previous version of Magento and Sample Data. In this case try to clean "./var" folder and browser cache and re-install Magento.

If the problem still exists could you please describe more details about how to reproduce the issue?

**Comment by voleye** [comment]
Hi @andrew-smart, if you have a chance, could you please look at the table "importexport_importdata" and describe which unique values contains the column "entity"?

P.S.
Did you use import/export capabilities before installing Sample Data?

**Comment by sivajik34** [comment]
Hi @voleye  I downloaded from magento site as tar folder.and we are trying to install from browser.at the time of 90% it stucks.There is a problem in configurable sample data.
I turned off configurable sample data module  then I’m able to install.

**Comment by andrew-smart** [comment]
@voleye There are no entries in that table.
I only downloaded the tar (with sample data), extracted and ran the installation through the web.

**Comment by saifmd4u** [comment]
I have downloaded latest and I am having exactly same issue stuck at 90%.

@sivajik34  Could you let me know how i could 'turned off configurable sample data module', so that i could continue the installation

**Comment by techsoulman** [comment]
I had the exact same problem(stopped at 10% the first time I tried, then 37%, then 66%, then 90% again and again and again) , I ended up reinstalling with the magento package that doesn't include the samples(first option on this page https://www.magentocommerce.com/download ). It worked really fast.

However I don't know if it is related but after installation I had this problem: https://github.com/magento/magento2/issues/2349

**Comment by cscd98** [comment]
Same issue here

**Comment by mcekutis** [comment]
I have simmilar issue, just getting more info:
Module 'Magento_ConfigurableSampleData':
Installing data.. 
[ERROR] Exception: Warning: Error while sending QUERY packet. PID=8102 in /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/zendframework1/library/Zend/Db/Statement/Pdo.php on line 228 in /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/framework/App/ErrorHandler.php:61
Stack trace:
#0 [internal function]: Magento\Framework\App\ErrorHandler->handler(2, 'Error while sen...', '/home/speedbad/...', 228, Array)
#1 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/zendframework1/library/Zend/Db/Statement/Pdo.php(228): PDOStatement->execute(Array)
#2 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/framework/DB/Statement/Pdo/Mysql.php(95): Zend_Db_Statement_Pdo->_execute(Array)
#3 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/zendframework1/library/Zend/Db/Statement.php(303): Magento\Framework\DB\Statement\Pdo\Mysql->_execute(Array)
#4 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/zendframework1/library/Zend/Db/Adapter/Abstract.php(480): Zend_Db_Statement->execute(Array)
#5 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/zendframework1/library/Zend/Db/Adapter/Pdo/Abstract.php(238): Zend_Db_Adapter_Abstract->query('UPDATE `setup_m...', Array)
#6 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/framework/DB/Adapter/Pdo/Mysql.php(444): Zend_Db_Adapter_Pdo_Abstract->query('UPDATE `setup_m...', Array)
#7 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/framework/DB/Adapter/Pdo/Mysql.php(499): Magento\Framework\DB\Adapter\Pdo\Mysql->_query('UPDATE `setup_m...', Array)
#8 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/zendframework1/library/Zend/Db/Adapter/Abstract.php(635): Magento\Framework\DB\Adapter\Pdo\Mysql->query('UPDATE `setup_m...', Array)
#9 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/magento/framework/Module/ModuleResource.php(129): Zend_Db_Adapter_Abstract->update('setup_module', Array, '(module = 'Mage...')
#10 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/setup/src/Magento/Setup/Model/Installer.php(808): Magento\Framework\Module\ModuleResource->setDataVersion('Magento_Configu...', '2.0.0')
#11 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/setup/src/Magento/Setup/Model/Installer.php(746): Magento\Setup\Model\Installer->handleDBSchemaData(Object(Magento\Setup\Module\DataSetup), 'data')
#12 [internal function]: Magento\Setup\Model\Installer->installDataFixtures()
#13 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/setup/src/Magento/Setup/Model/Installer.php(331): call_user_func_array(Array, Array)
#14 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/setup/src/Magento/Setup/Controller/Install.php(97): Magento\Setup\Model\Installer->install(Array)
#15 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/zendframework/zend-mvc/src/Controller/AbstractActionController.php(82): Magento\Setup\Controller\Install->startAction()
#16 [internal function]: Zend\Mvc\Controller\AbstractActionController->onDispatch(Object(Zend\Mvc\MvcEvent))
#17 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/zendframework/zend-eventmanager/src/EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#18 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/zendframework/zend-eventmanager/src/EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#19 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/zendframework/zend-mvc/src/Controller/AbstractController.php(118): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#20 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/zendframework/zend-mvc/src/DispatchListener.php(93): Zend\Mvc\Controller\AbstractController->dispatch(Object(Zend\Http\PhpEnvironment\Request), Object(Zend\Http\PhpEnvironment\Response))
#21 [internal function]: Zend\Mvc\DispatchListener->onDispatch(Object(Zend\Mvc\MvcEvent))
#22 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/zendframework/zend-eventmanager/src/EventManager.php(444): call_user_func(Array, Object(Zend\Mvc\MvcEvent))
#23 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/zendframework/zend-eventmanager/src/EventManager.php(205): Zend\EventManager\EventManager->triggerListeners('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#24 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/vendor/zendframework/zend-mvc/src/Application.php(314): Zend\EventManager\EventManager->trigger('dispatch', Object(Zend\Mvc\MvcEvent), Object(Closure))
#25 /home/speedbad/domains/ridikiuliai.lt/public_html/learn/setup/index.php(31): Zend\Mvc\Application->run()
#26 {main}

**Comment by sky4git** [comment]
Same issue: Magento 2 install stuck at 90% with sample data. 

Below is the install log: 
Starting Magento installation:
File permissions check...

Enabling Maintenance Mode...

Installing deployment configuration...

Installing database schema:
Schema creation/updates:
Module 'Magento_Store':
Installing schema..

Module 'Magento_AdvancedPricingImportExport':

Module 'Magento_Directory':
Installing schema..

Module 'Magento_Theme':
Installing schema..

Module 'Magento_Backend':

Module 'Magento_Backup':

Module 'Magento_Eav':
Installing schema..

Module 'Magento_Customer':
Installing schema.. Upgrading schema..

Module 'Magento_BundleImportExport':

Module 'Magento_AdminNotification':
Installing schema..

Module 'Magento_CacheInvalidate':

Module 'Magento_Indexer':
Installing schema..

Module 'Magento_Cms':
Installing schema..

Module 'Magento_CatalogImportExport':

Module 'Magento_Catalog':
Installing schema.. Upgrading schema..

Module 'Magento_Rule':

Module 'Magento_Msrp':

Module 'Magento_Search':
Installing schema.. Upgrading schema..

Module 'Magento_Bundle':
Installing schema.. Upgrading schema..

Module 'Magento_Quote':
Installing schema.. Upgrading schema..

Module 'Magento_CatalogUrlRewrite':
Installing schema..

Module 'Magento_Widget':
Installing schema..

Module 'Magento_SalesSequence':
Installing schema..

Module 'Magento_CheckoutAgreements':
Installing schema.. Upgrading schema..

Module 'Magento_Payment':

Module 'Magento_Downloadable':
Installing schema..

Module 'Magento_CmsUrlRewrite':

Module 'Magento_Config':
Installing schema..

Module 'Magento_ConfigurableImportExport':

Module 'Magento_CatalogInventory':
Installing schema..

Module 'Magento_SampleData':

Module 'Magento_Contact':

Module 'Magento_Cookie':

Module 'Magento_Cron':
Installing schema..

Module 'Magento_CurrencySymbol':

Module 'Magento_CatalogSearch':
Installing schema..

Module 'Magento_CustomerImportExport':

Module 'Magento_CustomerSampleData':

Module 'Magento_Deploy':

Module 'Magento_Developer':

Module 'Magento_Dhl':

Module 'Magento_Authorization':
Installing schema..

Module 'Magento_User':
Installing schema.. Upgrading schema..

Module 'Magento_ImportExport':
Installing schema.. Upgrading schema..

Module 'Magento_Sales':
Installing schema..

Module 'Magento_CatalogRule':
Installing schema..

Module 'Magento_Email':
Installing schema..

Module 'Magento_EncryptionKey':

Module 'Magento_Fedex':

Module 'Magento_GiftMessage':
Installing schema..

Module 'Magento_Checkout':

Module 'Magento_GoogleAnalytics':

Module 'Magento_GoogleOptimizer':
Installing schema..

Module 'Magento_GroupedImportExport':

Module 'Magento_GroupedProduct':

Module 'Magento_Tax':
Installing schema..

Module 'Magento_DownloadableImportExport':

Module 'Magento_Braintree':

Module 'Magento_Integration':
Installing schema..

Module 'Magento_LayeredNavigation':

Module 'Magento_Marketplace':

Module 'Magento_MediaStorage':

Module 'Magento_ConfigurableProduct':
Installing schema..

Module 'Magento_MsrpSampleData':

Module 'Magento_Multishipping':

Module 'Magento_NewRelicReporting':
Installing schema..

Module 'Magento_Newsletter':
Installing schema..

Module 'Magento_OfflinePayments':

Module 'Magento_SalesRule':
Installing schema..

Module 'Magento_OfflineShipping':
Installing schema..

Module 'Magento_PageCache':

Module 'Magento_Captcha':
Installing schema..

Module 'Magento_Paypal':
Installing schema..

Module 'Magento_Persistent':
Installing schema..

Module 'Magento_ProductAlert':
Installing schema..

Module 'Magento_Weee':
Installing schema..

Module 'Magento_ProductVideo':
Installing schema..

Module 'Magento_CatalogSampleData':

Module 'Magento_Reports':
Installing schema..

Module 'Magento_RequireJs':

Module 'Magento_Review':
Installing schema..

Module 'Magento_BundleSampleData':

Module 'Magento_Rss':

Module 'Magento_DownloadableSampleData':

Module 'Magento_Authorizenet':

Module 'Magento_OfflineShippingSampleData':

Module 'Magento_ConfigurableSampleData':

Module 'Magento_SalesSampleData':

Module 'Magento_ProductLinksSampleData':

Module 'Magento_ThemeSampleData':

Module 'Magento_ReviewSampleData':

Module 'Magento_SendFriend':
Installing schema..

Module 'Magento_Ui':
Installing schema..

Module 'Magento_Sitemap':
Installing schema..

Module 'Magento_CatalogRuleConfigurable':

Module 'Magento_Swagger':

Module 'Magento_Swatches':
Installing schema..

Module 'Magento_SwatchesSampleData':

Module 'Magento_GroupedProductSampleData':

Module 'Magento_TaxImportExport':

Module 'Magento_TaxSampleData':

Module 'Magento_GoogleAdwords':

Module 'Magento_CmsSampleData':

Module 'Magento_Translation':
Installing schema..

Module 'Magento_Shipping':

Module 'Magento_Ups':

Module 'Magento_UrlRewrite':
Installing schema..

Module 'Magento_CatalogRuleSampleData':

Module 'Magento_Usps':

Module 'Magento_Variable':
Installing schema..

Module 'Magento_Version':

Module 'Magento_Webapi':

Module 'Magento_SalesRuleSampleData':

Module 'Magento_CatalogWidget':

Module 'Magento_WidgetSampleData':

Module 'Magento_Wishlist':
Installing schema..

Module 'Magento_WishlistSampleData':

Schema post-updates:
Module 'Magento_Store':

Module 'Magento_AdvancedPricingImportExport':

Module 'Magento_Directory':

Module 'Magento_Theme':

Module 'Magento_Backend':

Module 'Magento_Backup':

Module 'Magento_Eav':

Module 'Magento_Customer':

Module 'Magento_BundleImportExport':

Module 'Magento_AdminNotification':

Module 'Magento_CacheInvalidate':

Module 'Magento_Indexer':
Running recurring..

Module 'Magento_Cms':

Module 'Magento_CatalogImportExport':

Module 'Magento_Catalog':

Module 'Magento_Rule':

Module 'Magento_Msrp':

Module 'Magento_Search':

Module 'Magento_Bundle':

Module 'Magento_Quote':

Module 'Magento_CatalogUrlRewrite':

Module 'Magento_Widget':

Module 'Magento_SalesSequence':

Module 'Magento_CheckoutAgreements':

Module 'Magento_Payment':

Module 'Magento_Downloadable':

Module 'Magento_CmsUrlRewrite':

Module 'Magento_Config':

Module 'Magento_ConfigurableImportExport':

Module 'Magento_CatalogInventory':

Module 'Magento_SampleData':

Module 'Magento_Contact':

Module 'Magento_Cookie':

Module 'Magento_Cron':

Module 'Magento_CurrencySymbol':

Module 'Magento_CatalogSearch':

Module 'Magento_CustomerImportExport':

Module 'Magento_CustomerSampleData':

Module 'Magento_Deploy':

Module 'Magento_Developer':

Module 'Magento_Dhl':

Module 'Magento_Authorization':

Module 'Magento_User':

Module 'Magento_ImportExport':

Module 'Magento_Sales':

Module 'Magento_CatalogRule':

Module 'Magento_Email':

Module 'Magento_EncryptionKey':

Module 'Magento_Fedex':

Module 'Magento_GiftMessage':

Module 'Magento_Checkout':

Module 'Magento_GoogleAnalytics':

Module 'Magento_GoogleOptimizer':

Module 'Magento_GroupedImportExport':

Module 'Magento_GroupedProduct':

Module 'Magento_Tax':

Module 'Magento_DownloadableImportExport':

Module 'Magento_Braintree':

Module 'Magento_Integration':

Module 'Magento_LayeredNavigation':

Module 'Magento_Marketplace':

Module 'Magento_MediaStorage':

Module 'Magento_ConfigurableProduct':

Module 'Magento_MsrpSampleData':

Module 'Magento_Multishipping':

Module 'Magento_NewRelicReporting':

Module 'Magento_Newsletter':

Module 'Magento_OfflinePayments':

Module 'Magento_SalesRule':

Module 'Magento_OfflineShipping':

Module 'Magento_PageCache':

Module 'Magento_Captcha':

Module 'Magento_Paypal':

Module 'Magento_Persistent':

Module 'Magento_ProductAlert':

Module 'Magento_Weee':

Module 'Magento_ProductVideo':

Module 'Magento_CatalogSampleData':

Module 'Magento_Reports':

Module 'Magento_RequireJs':

Module 'Magento_Review':

Module 'Magento_BundleSampleData':

Module 'Magento_Rss':

Module 'Magento_DownloadableSampleData':

Module 'Magento_Authorizenet':

Module 'Magento_OfflineShippingSampleData':

Module 'Magento_ConfigurableSampleData':

Module 'Magento_SalesSampleData':

Module 'Magento_ProductLinksSampleData':

Module 'Magento_ThemeSampleData':

Module 'Magento_ReviewSampleData':

Module 'Magento_SendFriend':

Module 'Magento_Ui':

Module 'Magento_Sitemap':

Module 'Magento_CatalogRuleConfigurable':

Module 'Magento_Swagger':

Module 'Magento_Swatches':

Module 'Magento_SwatchesSampleData':

Module 'Magento_GroupedProductSampleData':

Module 'Magento_TaxImportExport':

Module 'Magento_TaxSampleData':

Module 'Magento_GoogleAdwords':

Module 'Magento_CmsSampleData':

Module 'Magento_Translation':

Module 'Magento_Shipping':

Module 'Magento_Ups':

Module 'Magento_UrlRewrite':

Module 'Magento_CatalogRuleSampleData':

Module 'Magento_Usps':

Module 'Magento_Variable':

Module 'Magento_Version':

Module 'Magento_Webapi':

Module 'Magento_SalesRuleSampleData':

Module 'Magento_CatalogWidget':

Module 'Magento_WidgetSampleData':

Module 'Magento_Wishlist':

Module 'Magento_WishlistSampleData':

Installing user configuration...

Enabling caches:
Current status:
Array
(
[config] => 1
[layout] => 1
[block_html] => 1
[collections] => 1
[reflection] => 1
[db_ddl] => 1
[eav] => 1
[config_integration] => 1
[config_integration_api] => 1
[full_page] => 1
[translate] => 1
[config_webservice] => 1
)

Installing data...
Data install/update:
Module 'Magento_Store':

Module 'Magento_AdvancedPricingImportExport':

Module 'Magento_Directory':
Installing data..

Module 'Magento_Theme':
Installing data..

Module 'Magento_Backend':

Module 'Magento_Backup':

Module 'Magento_Eav':
Installing data..

Module 'Magento_Customer':
Installing data.. Upgrading data..

Module 'Magento_BundleImportExport':

Module 'Magento_AdminNotification':

Module 'Magento_CacheInvalidate':

Module 'Magento_Indexer':
Installing data..

Module 'Magento_Cms':
Installing data..

Module 'Magento_CatalogImportExport':

Module 'Magento_Catalog':
Installing data.. Upgrading data..

Module 'Magento_Rule':

Module 'Magento_Msrp':
Installing data..

Module 'Magento_Search':

Module 'Magento_Bundle':
Installing data..

Module 'Magento_Quote':
Installing data..

Module 'Magento_CatalogUrlRewrite':
Installing data..

Module 'Magento_Widget':
Installing data..

Module 'Magento_SalesSequence':
Installing data..

Module 'Magento_CheckoutAgreements':

Module 'Magento_Payment':

Module 'Magento_Downloadable':
Installing data..

Module 'Magento_CmsUrlRewrite':

Module 'Magento_Config':
Installing data..

Module 'Magento_ConfigurableImportExport':

Module 'Magento_CatalogInventory':
Installing data..

Module 'Magento_SampleData':
Installing data..

Module 'Magento_Contact':

Module 'Magento_Cookie':

Module 'Magento_Cron':

Module 'Magento_CurrencySymbol':

Module 'Magento_CatalogSearch':
Installing data..

Module 'Magento_CustomerImportExport':

Module 'Magento_CustomerSampleData':
Installing data..

Module 'Magento_Deploy':

Module 'Magento_Developer':

Module 'Magento_Dhl':
Installing data..

Module 'Magento_Authorization':
Installing data..

Module 'Magento_User':
Upgrading data..

Module 'Magento_ImportExport':

Module 'Magento_Sales':
Installing data.. Upgrading data..

Module 'Magento_CatalogRule':
Installing data..

Module 'Magento_Email':

Module 'Magento_EncryptionKey':

Module 'Magento_Fedex':
Installing data..

Module 'Magento_GiftMessage':
Installing data..

Module 'Magento_Checkout':
Installing data..

Module 'Magento_GoogleAnalytics':

Module 'Magento_GoogleOptimizer':

Module 'Magento_GroupedImportExport':

Module 'Magento_GroupedProduct':
Installing data..

Module 'Magento_Tax':
Installing data.. Upgrading data..

Module 'Magento_DownloadableImportExport':

Module 'Magento_Braintree':

Module 'Magento_Integration':

Module 'Magento_LayeredNavigation':

Module 'Magento_Marketplace':

Module 'Magento_MediaStorage':

Module 'Magento_ConfigurableProduct':
Installing data..

Module 'Magento_MsrpSampleData':
Installing data..

Module 'Magento_Multishipping':

Module 'Magento_NewRelicReporting':

Module 'Magento_Newsletter':

Module 'Magento_OfflinePayments':

Module 'Magento_SalesRule':
Installing data..

Module 'Magento_OfflineShipping':

Module 'Magento_PageCache':

Module 'Magento_Captcha':

Module 'Magento_Paypal':
Installing data..

Module 'Magento_Persistent':

Module 'Magento_ProductAlert':

Module 'Magento_Weee':
Installing data..

Module 'Magento_ProductVideo':
Upgrading data..

Module 'Magento_CatalogSampleData':
Installing data..

Module 'Magento_Reports':
Installing data..

Module 'Magento_RequireJs':

Module 'Magento_Review':
Installing data..

Module 'Magento_BundleSampleData':
Installing data..

Module 'Magento_Rss':

Module 'Magento_DownloadableSampleData':
Installing data..

Module 'Magento_Authorizenet':

Module 'Magento_OfflineShippingSampleData':
Installing data..

Module 'Magento_ConfigurableSampleData':
Installing data..

**Comment by mazhalai** [comment]
What are your values for `max_execution_time`? Please increase it to [18000](http://devdocs.magento.com/guides/v2.0/install-gde/trouble/php/tshoot_70pct.html)

**Comment by sky4git** [comment]
@mazhalai Its already 18000.

**Comment by lenkaren** [comment]
I have the similar problem.
It always  install stuck at 92% when install the sampledata.
The lastest I tried to install with the cms sampledata and it done.
But when it print "Success" information page,
it also said "Something went wrong while installing sample data. Please check var/log/system.log for details. You can retry installing the data now or just start using Magento."

then I check log.

[2015-12-07 08:19:36] main.ERROR: Sample Data error: No such entity with email = roni_cost@example.com, websiteId = 1 [] []
[2015-12-07 08:19:39] main.ERROR: Sample Data error: Notice: unserialize(): Error at offset 354 of 426 bytes in /home/wwwroot/magento2/vendor/magento/module-rule/Model/AbstractModel.php on line 183 [] []

**Comment by Vinai** [comment]
This thread has turned into a kitchen sink "Unable to install sample data" issue. The reason why seems to be different for most people contributing to this thread - at least the error messages are different.
This makes it really hard to keep track of the problems. What if the problem of the original poster is already fixed? Should the issue be closed? You see the problem.

So instead of "hijacking" an existing thread, please open a separate issue. If you want to say "my issue is similar to another one", please refer to the other issue by including the issue number in the post prefixing it with a `#` sign. Github will automatically cross-reference the issues then.

**Comment by Vinai** [comment]
@sivajik34 , is your issue still open? Are you still encountering the problem `Area code not set: Area code must be set before starting a session.` when installing the sample data?  
Also, have you installed Magento 2 using composer, by cloning the repository, or by downloading the ZIP file from the website?
If you cloned the repository, which approach did you take to install the sample data?

**Comment by vernard** [comment]
Hi all,

I have experienced this problem and found a working solution.

Bug Description:
There is no error message `install.log` (just stopped when installing data from `Magento_SalesSampleData`) , but the `system.log` displayed the following: 
`Sample Data error: Error in data structure: entity values are mixed [] []`

I am installing with sample data by [cloning repositories](http://devdocs.magento.com/guides/v2.0/install-gde/install/cli/install-cli-sample-data-clone.html)

Problem:
In my case, it was because of the line endings of the fixtures, specifically the 'attributes.csv' file from 'Magento_CatalogSampleData'. The option values in database have '\r' in them, which then crashes when the attribute option 'XS' is not found because the attribute option in database is 'XS\r'.

Solution:
On the repository `magento2-sample-data`, execute these commands:

```
git config core.autocrlf true
git rm *
git reset --hard HEAD
```

Then try running the setup again (clean installation). It should work now.

Update:

Using `git config core.autocrlf false` seems to work too. I'm not sure why but there are times the other still gives me the error.

**Comment by vkholoshenko** [comment]
@sivajik34, can you confirm that @vernard's solution  works for you?

**Comment by neeoon** [comment]
> What are your values for max_execution_time? Please increase it to 18000

helped for me

**Comment by Quotacious** [comment]
Yes, I'm also continuing the thread hijack, because there is nothing else coming up in Google Search.

I found the following advice on another site:

Change the below values in php.ini file to:
(if you are using php5-fpm, make sure you edit the correct php5.ini file)

max_execution_time = 1800
max_input_time = 1800
memory_limit = 1024M

If you are using php5-fpm, be sure to restart it or the memory_limit change will not take effect.

The changes above also require one really important thing, PATIENCE. Why?  The progress bar zooms along to 87% and then ticks up to 90% where it seems to stop dead.  Your instinct is that it's failed and you start troubleshooting.  The answer is, it's slowly copying sample data into place but failing to update the progress bar.  Wait long enough and it should advance to 91% and a while after that, the glorious 100% is reached.

I had to reinstall about 5 times before finally getting this to work; I know this bug is hijacked, but after finally getting it to work, I had to at least pass it along.

**Comment by vzabaznov** [comment]
@sivajik34 looks like problem could be solved with local git setting, since there is no updates from 21 Jan, i close issue, if you feel that issue is not fixed or you have another question please feel free to write here or open another issue

**Comment by alihalabyah** [comment]
For anyone still facing this issue, apparently it is related to the permissions on the log files, Magento keeps trying to write whatever log before showing the successful page, therefore just give the correct permissions while the indicator still showing 90% and on the next poll request you'll get the success page

**Comment by mazhalai** [comment]
@alihalabyah please do not post on closed issues, create a new one.

**Comment by lojasmarques** [comment]
Why I need to loose night-after-night of sleep to try solve that bug. it is not working and nobody knows how to solve the 339/501 Módulo 'Magento_ConfigurableSampleData': 
Instalando dados ... Until the next Millennium!
If someone can help, please contact my telephone number (5553) 981426029 [ WhatsApp ] or by email suporte@lojasmarques.com
I lost three weeks in this damn problem

**Comment by Ctucker9233** [comment]
@lojasmarques There are several comments above that propose solutions. If none of those work for you, open a new issue. No one's going to be able to help if you don't describe your issue clearly.

**Comment by ghost** [comment]
When you encounter this problem, restart your PHP and APACHE (NGINX) service, and then try again and you'll find that everything is ok.

**Comment by vibrantBits** [comment]
This software is crap. It stucks at every step.

Can't believe it is a market leader.
